project_one/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Cool little thing</title>
+
+    <link rel="stylesheet" type="text/css" href="project1.css">
+</head>
+<body>
+
+    <script type="text/javascript">
+        const clicked  = () => {
+        let kid = document.createElement("div");
+        kid.setAttribute("id", "grading-element-one-consequence");
+        kid.appendChild(document.createTextNode("S H I T"));
+        document.body.appendChild(kid);
+        }
+
+
+
+    </script>
+
+
+    <h1>
+    	Pretty Patties <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/28/Supreme_Logo.svg/1200px-Supreme_Logo.svg.png" width="250" height="64">
+    </h1>
+  <!--   This is section 1 -->
+    <div id="Text1">
+        <p>
+            <h1>
+                This is section one
+                <button id="grading-element-one" onclick="clicked()">Press me</button>
+            </h1>
+
+        </p>
+    </div>
+
+    <!-- This is section 2 -->
+    <div id="Text2">
+        <p>
+            <h2>
+                <a href="file:///Users/edisonblum/Documents/sublime/about.html" id="grading-element-two">About</a>
+            </h2>
+
+        </p>
+    </div>
+
+    <!-- This is section 3 -->
+    <div id="Text3">
+        <p>
+        	<h2>
+        		This is section three
+            	<button id="grading-element-three" 
+                    onclick="getElementById('grading-element-three-consequence').innerHTML = 'this should work'">
```

Comment:
effectively similar, but not exactly what I asked for. The text can already be there...you need to start the element as `visibility: hidden;` then on click, change it to `visibility: visible`